<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HSQVMDT" failures="2" tests="2" name="Test" time="0.033" errors="0" timestamp="11 Dec 2020 04:22:24 GMT">
  <testcase classname="TestNG_Examples.softAssertion" name="hardAssertion" time="0.01">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertNull(Assert.java:233)
	at junit.framework.Assert.assertNull(Assert.java:226)
	at TestNG_Examples.softAssertion.hardAssertion(softAssertion.java:20)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- hardAssertion -->
  <testcase classname="TestNG_Examples.softAssertion" name="softAsssertion" time="0.004">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [null] but found [assertion]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [null] but found [assertion]
	at TestNG_Examples.softAssertion.softAsssertion(softAssertion.java:34)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- softAsssertion -->
</testsuite> <!-- Test -->
